### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Uthmanbello**
In this project:
1. I prepared pop-up windows for mobile and desktop views
2. I created an object in JavaScript to store the information used for the pop-ups
3. I used linters to check for errors

### CODE REVIEW

**Review by OyePriscilla — COMMENTED** [review]
Hi @Uthmanbello 👍 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that ought to be pair programing, kindly get across to your partner(s) for his/her commits.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -86,8 +87,8 @@ <h2 class="head2">Tonic</h2>
                         <div class="link3">javaScript</div>
                     </li>
                 </ul>
-                <button class="see-project1 unique2">
-                    <div class="see-project">See Project</div>
+                <button class="see-project1 project-buttons unique2">
+                    <div class="see-project" id="sec2-project">See Project</div>
```
Comment (fobadara) [inline index.html]: [REQUIRED] None of your popups is working. No matter how much I click the `see project` buttons I get nothing.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Uthmanbello) [reply index.html]: @fobadara Hello,

They are all actually working. I just tried them now; both in mobile and desktop view.

image

File: index.html [hunk 2]
```diff
@@ -86,8 +87,8 @@ <h2 class="head2">Tonic</h2>
                         <div class="link3">javaScript</div>
                     </li>
                 </ul>
-                <button class="see-project1 unique2">
-                    <div class="see-project">See Project</div>
+                <button class="see-project1 project-buttons unique2">
```
Comment (huseyinbiyik) [inline index.html]: Hello team! Good job, but please make sure all the buttons are working properly as the previous reviewer requested, and make sure you created all cards created dynamically.
Screen Recording 2022-12-24 at 1 00 03 AM

[OPTIONAL]  According to the project requirement [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20use,when%20the%20page%20loads.), the lines between 60-228 need to be removed and should be generated dynamically with JavaScript.

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
tenor

 ## Highlights
- Linter checks pass 🟢
- An array created including projects data 👌


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Uthmanbello) [reply index.html]: @huseyinbiyik Hello,

Lines 60 - 228 are for the normal window web page and not for the pop-up window. Am I expected to remove them too?

Comment (huseyinbiyik) [reply index.html]: Hello,
No only the cards in the middle. You need to have a container wrapping the cards but all the rest need to be created dynamically with JavaScript. You can check my portfolio if you need an example 👍

**Review by fobadara — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ linters are passing

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**